code/modules/uplink/uplink_items.dm

```diff
@@ -1786,6 +1786,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/storage/box/hug/reverse_revolver
 	restricted_roles = list("Clown")
 
+/datum/uplink_item/role_restricted/laser_arm
+	name = "Laser arm implant"
+	desc = "A laser arm implant which grants you with a functionally antique lasergun technology inside of your arm, Keep away from emps. Comes with a autosurgeon."
```

Comment:
This reads in a way that's awkward.
"(...) grants you a lasergun, functionally similar to the antique laser, inside of your arm (...)" would read much better.

Reply:
sorry i missed grammar class

Reply:
This is a genuine problem, please consider changing this.

Reply:
it's clearly a reference to the poor english translations of some of the mega man games, smh my head my head